packages/frontend/src/providers/DashboardProvider.tsx

```diff
@@ -570,6 +575,38 @@ export const DashboardProvider: React.FC<
         [resultsCacheTimes],
     );
 
+    // Filters that are required to have a value set
+    const requiredDashboardFilters = useMemo(
+        () =>
+            dashboardFilters.dimensions
+                .filter((f) => f.required && f.disabled)
```

Comment:
what is the difference between required and disabled?

Reply:
disabled: no default value set / is any
required: requires a value to be set

Reply:
https://github.com/lightdash/lightdash/pull/9505/commits/7fbe5bf65a3ca1514f87ceb22320ac48bf9957fe